src/Core_Sideloader/Core.Sideloader.cs

```diff
@@ -13,10 +13,12 @@
 using System.Linq;
 using System.Reflection;
 using System.Text;
+using BepisPlugins;
 using UnityEngine;
 using XUnity.ResourceRedirector;
 #if AI || HS2
 using AIChara;
+
```

Comment:
Please avoid unnecessary changes since it's harder to see what parts actually matter and it messes with git history

Reply:
Oof okay

Reply:
https://github.com/IllusionMods/BepisPlugins/pull/122/commits/a57e34b7c5f0c984a134ea1e0023d8c6bfee7d86